perma_web/perma/migrations/0028_auto_20230905_1813.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.2.20 on 2023-09-05 18:13
+
+from django.db import migrations, models
+
+
+def fixup_private_reason(apps, schema_editor):
+    """
+    Instead of 'policy' and 'old_policy', use descriptive keys
+    """
+    Link = apps.get_model('perma', 'Link')
+    print("Migrating 'old_policy' to 'meta_generic'.")
```

Comment:
```suggestion
    print("Migrating 'old_policy' to 'meta'.")
```
unless you meant to use `meta_generic`?

Reply:
Thanks. A first draft of this used longer key names... but it turned out that migrating the column to allow more than 10 chars was expensive, so I decided not worth it.